src/main/java/com/spotinst/sdkjava/model/api/azure/elastiGroup/V3/ApiNewVmsAzure.java

```diff
@@ -0,0 +1,182 @@
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiNewVmsAzure implements IPartialUpdateEntity {
+
+    @JsonIgnore
+    private Set<String> isSet;
+    private String      createdAt;
+    private String      lifeCycle;
+    private String      os;
+    private String      powerState;
+    private Integer     privateIp;
+    private String      provisioningState;
+    private Integer     publicIp;
+    private String      region;
+    private String      vmName;
+    private String      vmSize;
+
+    public ApiNewVmsAzure() {
+        isSet = new HashSet<>();
+    }
+
+    public Set<String> getIsSet() {
+        return isSet;
+    }
+
+    public void setIsSet(Set<String> isSet) {
+        this.isSet = isSet;
+    }
+
+    public String getCreatedAt() {
+        return createdAt;
+    }
+
+    public void setCreatedAt(String createdAt) {
+        isSet.add("createdAt");
+        this.createdAt = createdAt;
+    }
+
+    public String getLifeCycle() {
+        return lifeCycle;
+    }
+
+    public void setLifeCycle(String lifeCycle) {
+        isSet.add("lifeCycle");
+        this.lifeCycle = lifeCycle;
+    }
+
+    public String getOs() {
+        return os;
+    }
+
+    public void setOs(String os) {
+        isSet.add("os");
+        this.os = os;
+    }
+
+    public String getPowerState() {
+        return powerState;
+    }
+
+    public void setPowerState(String powerState) {
+        isSet.add("powerState");
+        this.powerState = powerState;
+    }
+
+    public Integer getPrivateIp() {
+        return privateIp;
+    }
+
+    public void setPrivateIp(Integer privateIp) {
+        isSet.add("privateIp");
+        this.privateIp = privateIp;
+    }
+
+    public String getProvisioningState() {
+        return provisioningState;
+    }
+
+    public void setProvisioningState(String provisioningState) {
+        isSet.add("provisioningState");
+        this.provisioningState = provisioningState;
+    }
+
+    public Integer getPublicIp() {
+        return publicIp;
+    }
+
+    public void setPublicIp(Integer publicIp) {
+        isSet.add("publicIp");
+        this.publicIp = publicIp;
+    }
+
+    public String getRegion() {
+        return region;
+    }
+
+    public void setRegion(String region) {
+        isSet.add("region");
+        this.region = region;
+    }
+
+    public String getVmName() {
+        return vmName;
+    }
+
+    public void setVmName(String vmName) {
+        isSet.add("vmName");
+        this.vmName = vmName;
+    }
+
+    public String getVmSize() {
+        return vmSize;
+    }
+
+    public void setVmSize(String vmSize) {
+        isSet.add("vmSize");
+        this.vmSize = vmSize;
+    }
+
+
+    @JsonIgnore
+    public boolean isCreatedAtSet() {
+        return isSet.contains("createdAt");
+    }
+
+    @JsonIgnore
+    public boolean isLifeCycleSet() {
```

Comment:
convert boolean to Boolean in all files

Reply:
most of the API classes we are using boolean instead of Boolean and few of the API sdk were already implemented for this, hence continued with same